hs_core/templates/search/search.html

```diff
@@ -149,62 +149,100 @@ <h4 class="panel-title">
-                            {% endif %}&laquo; Previous
-                            {% if page_obj.has_previous %}</a>{% endif %}
-                            |
-                            {% if page_obj.has_next %}
-                                <a href="{{ request.get_full_path|clean_pagination_url }}&amp;page={{ page_obj.next_page_number }}">
-                            {% endif %}Next &raquo;{% if page_obj.has_next %}</a>{% endif %}
+                            </thead>
+                            <tbody id="items-discovered-tbody">
+                                {% for result in page_obj.object_list %}
+                                    <tr>
+                                        <td><a href="{{ result.object.get_absolute_url }}">{{ result.object.metadata.title.value }}</a></td>
+                                        <td>{{ result.object|resource_type }}</td>
+                                        {% if result.object.first_creator.description %}
+                                            <td><a href="{{ result.object.first_creator.description }}">{{ result.object.first_creator.name  }}</a></td>
+                                        {% else %}
+                                            <td>{{ result.object.first_creator.name }}</td>
+                                        {% endif %}
+                                        <td>{{ result.object.updated|date }} {{ result.object.updated|time }}</td>
+                                    </tr>
+                                {% empty %}
+                                    <p>No results found.</p>
+                                {% endfor %}
+                            </tbody>
+                        </table>
+                        <div id="page-options" style="float: right">
+                        {% if page_obj.has_previous or page_obj.has_next %}
+                                {% if page_obj.has_previous %}
+                                    <a href="{{ request.get_full_path|clean_pagination_url }}&amp;page={{ page_obj.previous_page_number }}">
+                                {% endif %}&laquo; Previous
+                                {% if page_obj.has_previous %}</a>{% endif %}
+                                |
+                                {% if page_obj.has_next %}
+                                    <a href="{{ request.get_full_path|clean_pagination_url }}&amp;page={{ page_obj.next_page_number }}">
+                                {% endif %}Next &raquo;{% if page_obj.has_next %}</a>{% endif %}
+                        {% endif %}
                         </div>
-                    {% endif %}
-                </div>
-                <div id="map-view" class="tab-pane fade">
-                    <div id="discover-map" style="width:850px; height:650px"></div>
+                    </div>
+                    <div id="map-view" class="tab-pane fade">
+                        <div id="discover-map" style="width:850px; height:650px"></div>
```

Comment:
@xuezhaokun Can you try to clear all cache from your web browser to see if it work? I did run into this from my linux browser and after I cleared all cache, it worked, so hopefully it will work for you as well.